### DISCUSSION TITLE
How do I set multiple entry points for webpack in next.config.js?

### DISCUSSION THREAD

**Discussion by didimedina** [vercel/next.js#33400] [Help]
I'm trying to get webpack to compile one lib before the rest. From what I understand, in webpack5 you can do that by setting multiple entries like [this](https://survivejs.com/webpack/building/bundle-splitting/#bundle-splitting-at-entry-configuration): 
```Javascript
const config = {
  entry: {
    app: {
      import: path.join(__dirname, "src", "index.js"),
      dependOn: "vendor",
    },
    vendor: ["react", "react-dom"],
  },
};
``` 

Based on the little information, I found here on [NextJS docs](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config) and the web, I assumed something like this would work in the context of my project in the `next.config.js` file: 
```Javascript
/** @type {import('next').NextConfig} */

const path = require("path");

module.exports = {
  reactStrictMode: true,
  webpack: (config, options) => {

    const newWebpackConfig = {
      entry: {
        app: {
          import: path.join(__dirname, "pages", "_app.tsx"),
          dependOn: "vendor",
        },
        vendor: ["tailwindcss"],
      }
    }

    console.log(options.webpack)
    Object.assign(options.webpack, newWebpackConfig)

    return config
  },
}
```

But running into errors when I run `npm run dev` ... what's the proper way to set multiple entry points for a next js using webpack5?

My desired outcome is that Tailwind compiles before the rest of my packages. Would appreciate any help I can get on this 🙌🏼

**Comment by yunsii** [comment]
https://github.com/vercel/next.js/issues/774#issuecomment-272691895 This is work for me.
